Global database and summary sheet.R

```diff
@@ -51,92 +61,79 @@ write.csv(globalrisk, "Risk_sheets/Global_compound_risk_database.csv")
 #
 ##
 ### ********************************************************************************************
-####    CREATE A FLAG SUMMARY SHEET
+####    CREATE A FLAG SUMMARY SHEET ----
 ### ********************************************************************************************
 ##
 #
 
 # Add existing and emerging risk scores
+#—`riskflags` ----
 riskflags <- globalrisk %>%
```

Comment:
All edits in this function are to change the indicators or variables used in the model. Indicators have been dropped, renamed or added.